source/primer/devops/continuous_integration.rst

```diff
@@ -0,0 +1,117 @@
+Continuous Integration
+=====================
+
+.. admonition:: :fa:`graduation-cap` Learning Objectives
+
+    .. include:: objectives/continuous_integration.rst
+
+In the context of Tudat, Continuous Integration is used to streamline the development process.
+Thanks to it, developers need not worry about every step required to build and release the conda packages and documentation.
+Instead, they can either wait for nightly releases that are made automatically, or add specific tags in a commit message and let everything else run hidden trough :ref:`GitHub<GitHub actions>`, :ref:`Azure<Azure>`, :ref:`Anaconda<Anaconda>`, and :ref:`Readthedocs<Readthedocs>`.
+
+GitHub actions
+--------------
+.. image:: https://github.com/tudat-team/.github/actions/workflows/webservices.yml/badge.svg
+    :target: https://github.com/tudat-team/.github/actions/workflows/webservices.yml
+
+GitHub actions are integrated in Tudat development process to ease and automate the build and release of conda packages.
+
+Most importantly, the `webservices-dispatch-action repository <https://github.com/tudat-team/webservices-dispatch-action>`_ contains code that is used to trigger a build of a conda package.
+More information on how this action works is given in that repo's `README <https://github.com/tudat-team/webservices-dispatch-action/blob/main/README.md>`_.
```

Comment:
The readme mainly gives information on what it does, and less on how it works. Could you add (here or in the README) some details of what each file in the webservices repo does, and how exactly its interaction with other repos works?

Reply:
I added more information on this in the webservices-dispatch-action README with [9150bd4](https://github.com/tudat-team/webservices-dispatch-action/commit/9150bd4e162cc5526bbbe179a927a3120b269730), in new `Repo structure` and `Communication with other repositories` sections.

@geoffreygarrett could you, when you have time, take a look at the [Communication with other repositories](https://github.com/tudat-team/webservices-dispatch-action#communication-with-other-repositories) section? I think that the order in which the actions communicate with each other may be incorrect. And I believe that this setup is clearer to you than it is to me.